keyboards/kaly/kaly42/info.json

```diff
@@ -0,0 +1,92 @@
+{
+    "manufacturer": "Dwctor",
+    "keyboard_name": "kaly/kaly42",
+    "maintainer": "Dwctor",
+    "development_board": "blackpill_f401",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "direct": [
+            ["B1", "B10", "A8", "B15", "B14", "B13"],
+            ["B9", "B8", "B5", "B4", "B3", "A15"],
+            ["A4", "A3", "A5", "A6", "A7", "B0"],
+            ["A2", "A1", "A0", "NO_PIN", "NO_PIN", "NO_PIN"]
+        ]
+    },
+    "split": {
+        "enabled": true,
+        "bootmagic": {
+            "matrix": [4, 5]
+        },
+        "matrix_pins": {
+            "right": {
+                "direct": [
+                    ["B13", "B14", "B15", "A8", "B10", "B1"],
+                    ["A15", "B3", "B4", "B5", "B8", "B9"],
+                    ["B0", "A7", "A6", "A5", "A3", "A4"],
+                    ["A0", "A1", "A2", "NO_PIN", "NO_PIN", "NO_PIN"]
+                ]
+            }
+        }
+    },
+    "url": "https://github.com/Dwctor/Kaly",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0000",
+        "vid": "0xFEED"
+    },
```

Comment:
```suggestion
    },
    "community_layouts": ["split_3x6_3"],
```

Reply:
Sorry about this one! I had added this in another version and forgot to re-add it to this cleaner one, thanks for catching it!